README.md

```diff
@@ -1,4 +1,7 @@
 # Roll For Reaction
+
+<img src="https://github.com/Matteas-Eden/roll-for-reaction/blob/master/public/logo.png" width="400">
+
```

Comment:
I'd prefer it if we used a link that isn't subject to change, like the one posted in a comment on the issue.

`![logo](https://user-images.githubusercontent.com/45865186/77014224-a9265180-69d6-11ea-92d3-18af1d76c740.png)`